internal/metrics/metrics.go

```diff
@@ -248,6 +250,28 @@ func (t *TSDB) MaxTime() time.Time {
 	return fromDbTime(t.db.Head().MaxTime())
 }
 
+// HeadMinTime
+func (t *TSDB) HeadMinTime() time.Time {
+	return fromDbTime(t.db.Head().MinTime())
+}
+
+// BlockMinMax
```

Comment:
I would add a comment on this function, not sure what is achieved here.

Reply:
fixed